src/utils/normalization.ts

```diff
@@ -107,3 +107,30 @@ export const normalize = (payload: OperationsRepository) => {
 
   return { apis, documentation } as OperationsRepository;
 };
+
+export const emptyObject = (object: any, preserveValueKeys: string[], ignoreNestedKeys: string[]) => {
```

Comment:
I was thinking it was okay to put this here but feel free to suggest somewhere else